src/libs/filter-utils.ts

```diff
@@ -10,23 +10,25 @@ export namespace FilterUtilities {
    * @returns If specified dates were null or not, if specified date parameters were between Forecast dates or if currentDate is between Forecast dates
    */
   export const filterByDate = (dateRange: DateRange, currentDate: Date, parameters: { startDate?: Date, endDate?: Date }): boolean => {
-    if (dateRange.start_date === null || dateRange.end_date === null) {
+    if (dateRange.start_date === null) {
       return false;
     }
-
+    const start_date = new Date(dateRange.start_date);
+    let end_date = new Date(dateRange.end_date);
+    if (dateRange.end_date === null) end_date = currentDate;
     if (parameters.startDate) {
-      if (parameters.startDate <= new Date(dateRange.start_date)) {
+      if (parameters.startDate <= start_date) {
```

Comment:
Rather than these nested ifs I believe you could use condition:
(parameters.startDate && parameters.startDate <= start_date)

Perhaps you could even use (parameters?.startDate <= start_date)
but you should check the filter still works as intended